src/ApiAlerts.Common/ApiAlerts.Common.csproj

```diff
@@ -23,7 +23,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="ApiUtilities.Common" Version="1.0.9" />
+    <PackageReference Include="ApiUtilities.Common" Version="1.0.10" />
```

Comment:
Are we sure we wanted to update this package?
This isn't API alerts this is APIUtilities a package i have on Nuget :)